systests/microprofile/client/weld/pom.xml

```diff
@@ -178,7 +188,7 @@
                     <dependency>
                         <groupId>com.github.tomakehurst</groupId>
                         <artifactId>wiremock-jre8-standalone</artifactId>
-                        <version>2.35.1</version>
+                        <version>3.0.1</version>
```

Comment:
The 2.35.x line was used for Jetty 9 support, if we don't need that, let's use the latest one (please notice group change, `org.wiremock` now):

```
<dependency>
    <groupId>org.wiremock</groupId>
    <artifactId>wiremock-standalone</artifactId>
    <version>3.3.1</version>
    <scope>test</scope>
</dependency>
```

Reply:
a new commit addressed it.

And thanks for the review and input @reta !